cmd/root.go

```diff
@@ -30,15 +30,19 @@ func NewMethodK8s(version string) *MethodK8s {
 	methodK8s := MethodK8s{
 		version: version,
 		RootFlags: config.RootFlags{
-			Quiet:          false,
-			Verbose:        false,
-			Context:        "",
-			Path:           "",
-			URL:            "",
-			ServiceAccount: false,
-			Token:          "",
-			CACert:         "",
-			APIServerURL:   "",
+			Quiet:   false,
+			Verbose: false,
+			ServiceAccountConfig: config.ServiceAccountConfig{
+				ServiceAccount: false,
+				Token:          "",
+				CACert:         "",
+				APIServerURL:   "",
+			},
+			OtherConfig: config.OtherConfig{
```

Comment:
Appreciate breaking into structs, but I'm very much one of those folks that thinks naming things is a super important part of engineering, so let's spend some time to have a better name for this. 

Given that it's holding information about the kubeconfig, why don't we have it be something like "KubeConfig".